### PR TITLE
chore: Add test coverage to events utils

### PR DESCRIPTION
**Pull request by SCasarotto**
Follow on to #1120 for events.

### CODE REVIEW

File: packages/ariakit-utils/src/__tests__/events-test.ts [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+  addGlobalEventListener,
+  fireBlurEvent,
+  fireClickEvent,
+  fireEvent,
+  fireKeyboardEvent,
+  isFocusEventOutside,
+  isPortalEvent,
+  isSelfTarget,
+  queueBeforeEvent,
+} from "ariakit-utils/events";
```
Comment (diegohaz) [inline packages/ariakit-utils/src/__tests__/events-test.ts]:
```suggestion
} from "../events";
```

**Review by diegohaz — COMMENTED** [review]
Awesome @SCasarotto!

I wonder if we should use `ariakit-test-utils`' render function in these tests to avoid having to clean up the DOM.

**Comment by SCasarotto** [comment]
> Awesome @SCasarotto!
> 
> I wonder if we should use `ariakit-test-utils`' render function in these tests to avoid having to clean up the DOM.

Looking at the `render` function, I noticed that it is just `@testing-library`'s function. Looks like it does automatically clean up. We could do that in both of these testing PRs. Do you think that is better? If so I can update it on both. 

I do wonder if I switch to `render` if I should start to use the `@testing-library`'s selectors as well. Thoughts?

**Comment by diegohaz** [comment]
I think it's worth using `ariakit-test-utils` on these tests and the selectors since they will simplify the code a lot.

I'll merge this PR as it's in a good state. But feel free to open a follow-up PR if you think the change above makes sense.

**Review by diegohaz — APPROVED** [review]

**Pull request merged**